practice-two/src/style/sections/testinomials.css

```diff
@@ -0,0 +1,175 @@
+.testimonials {
+  background-image: url("../../assets/images/background-testimonial.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.testimonials__wrapper {
+  padding-bottom: 30px;
+}
+
+.testimonials__header-text {
+  text-align: center;
+  padding-top: 28px;
+}
+
+.testimonials__description {
+  position: relative;
+  padding-top: 56px;
+  width: 240px;
+}
+
+.testimonials__description-text {
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-base);
+}
+
+.testimonials__description::before {
+  content: '"';
+  font-size: var(--font-size-3xlg);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-normal);
+  color: var(--color-primary);
+  position: absolute;
+  left: 50%;
+  top: 10%;
+  transform: translateX(-50%);
+}
+
+.testimonials__item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.testimonials___info-img {
+  border-radius: 50%;
+}
+
+.testimonials___info {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.testimonials__contact {
+  display: flex;
+  flex-direction: column;
+  margin-left: 10px;
+  gap: 5px;
+}
+
+.testimonials__link {
+  display: none;
+}
+
+.testimonials__contact-name {
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+}
+
+.testimonials__contact-mail {
+  font-size: var(--font-size-2xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+}
+
+@media (min-width: 768px) {
+  .testimonials__wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .testimonials__link {
+    display: block;
+
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+
+  .testimonials__header {
+    margin-top: 40px;
+  }
+
+  .testimonials__content {
+    display: flex;
+    align-items: center;
+    gap: 40px;
+    margin-top: 24px;
+  }
+
+  .testimonials__description {
+    width: 470px;
+  }
+
+  .testimonials__description-text {
```

Comment:
No need to override this style here. Because you can define style for it in mobile